File: 7class/Tema6/ex1.py
using namespace std;

int main() {
    int a, d, e, f, des, b, c;
    
    cin >> a >> d >> e >> f;
    
   if (f - d >= 0) {b = f - d; des = 0;} else {b = f + 10 - d; des = 1;}
   if (des == 0) c = e - a; else c = e - a - 1;
   cout << b << " " << c;
}

File: 7class/Tema10/ex1.cpp
using namespace std;

int main() {

  int n, i;
  cin >> n;
  for (i=1; i<=n; i++)
   {
      if (n%i==0) 
       {
           cout << i;
           if (i<n) cout << " ";
       }   
   }
  cout << endl;
  for (i=n; i>=1; i--)
   {
      if (n%i==0) 
       {
           cout << i;
           if (i>1) cout << " ";
       }   
}